src/components/split-button/bl-split-button.css

```diff
@@ -0,0 +1,70 @@
+:host {
+  position: relative;
+  display: inline-block;
+}
+
+:host([kind="neutral"]) bl-popover {
+  --bl-popover-border-color: var(--bl-color-neutral-darker);
+}
+
+:host([kind="success"]) bl-popover {
+  --bl-popover-border-color: var(--bl-color-success);
+}
+
+:host([kind="danger"]) bl-popover {
+  --bl-popover-border-color: var(--bl-color-danger);
+}
+
+.split-button-container {
+  display: flex;
+}
+
+.split-main-button {
+  width: 100%;
+
+  --bl-border-radius-m: calc(var(--bl-size-xs) / 2) 0 0 calc(var(--bl-size-xs) / 2);
+}
+
+.split-main-button:focus {
+  --bl-border-radius-l: calc(var(--bl-size-m) / 2) 0 0 calc(var(--bl-size-m) / 2);
+}
+
+.dropdown-button {
+  --bl-border-radius-m: 0 calc(var(--bl-size-xs) / 2) calc(var(--bl-size-xs) / 2) 0;
+}
+
+.dropdown-button:focus {
+  --bl-border-radius-l: 0 calc(var(--bl-size-m) / 2) calc(var(--bl-size-m) / 2) 0;
+}
+
+:host([variant="secondary"]) .dropdown-button {
+  left: -1px;
```

Comment:
is there an alternative method besides using negative values?

Reply:
Frankly, I couldn't find another method. Since in the variant(secondary), the main button and the dropdown button border are sides by side, it spoils the design like the image below. We discussed not adding CSS properties about the border to the bl-button element. So I couldn't find any other solution. I'd like to hear if you have a solution to recommend.


<img width="172" alt="Screenshot 2024-03-27 at 09 43 59" src="https://github.com/Trendyol/baklava/assets/66769855/e36ef65d-1bdc-4a15-826b-435a1c2f7ebd">

Reply:
Not a problem at all. Thank you for your hard work! 🚀